src/components/AddressSearch/index.js

```diff
@@ -254,6 +255,30 @@ function AddressSearch(props) {
             values.state = stateFallback;
         }
 
+        // Some edge-case addresses may lack both street_number and route in the API response, resulting in an empty "values.street"
+        // We are setting up a fallback to ensure "values.street" is populated with a relevant value
+        if (!values.street) {
+            const {adr_address} = details;
+            const streetAddressRegex = /(<span\s+class\s*=\s*"street-address"[^>]*>[^<]*<\/span>)/;
+            const streetAddressHtml = adr_address.match(streetAddressRegex);
+            if (streetAddressHtml[1]) {
```

Comment:
This will be undefined if there's no match

Reply:
This is why I kept the else code, but supposidly google API will always return adr_address in that format

Reply:
I mean it will throw an error, since if it fails `match` will return null and not an array.

Reply:
Yeah you're right, I'll add a catch of some sort or change something else here soon

Reply:
Sure, no rush - I just wanted to give it a quick look over. I don't have time to run all the tests today anyway :smile: